core/Cargo.toml

```diff
@@ -120,3 +124,4 @@ snowflake-connector-rs = { git = "https://github.com/dust-tt/snowflake-connector
 axum-test = "16.0.0"
 csv = "1.3.0"
 tikv-jemallocator = "0.6"
+elasticsearch = "8.15.0-alpha.1"
```

Comment:
why alpha?

Reply:
the official lib is only available in alpha
not too scary though since it's a somewhat direct curl wrapper